src/guides/v2.3/install-gde/install/cli/install-cli-subcommands-maint.md

```diff
@@ -85,6 +85,42 @@ where
 
 `--none` clears the list.
 
+## For Multistore setups {#instgde-cli-multistore}
+
+When we need different layout and localized content, general speculation is that there is not an out of the box feature. But, there is one. The error processor accepts an parameter to change the error page template.
+
+```php
```

Comment:
Where does this PHP come from?

Reply:
There was a lint error that was indicating that I have to specify the type of the language I used here. That's why I had to add that php there

Reply:
What I meant was, where does the PHP code come from? You haven't explained what file and the context